hca/upload/upload_area.py

```diff
@@ -58,3 +62,10 @@ def forget(self):
     def list(self):
         client = ApiClient(self.urn.deployment_stage)
         return client.list_area(self.uuid)
+
+    def upload_file(self, file_path, target_filename=None, report_progress=False, dcp_type=None):
```

Comment:
super minor nit: would prefer content_type/dcp_type to be the earlier argument (as opposed to `report_progress`), kind of like how s3agent does it.  it's a more important argument.

Reply:
👍